deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +254,143 @@ const awsConfig = new AWSConfig({
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent1 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
+}
+
+export function sendEvent2 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[1])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[1])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[1])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent2 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
+}
+
+export function sendEvent3 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[2])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[2])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[2])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent3 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
+}
+
+export function sendEvent4 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[3])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[3])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[3])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent4 === debug === messageBody \n', messageBody)
```

Comment:
```suggestion
```
Remove `console.log()` debugging, this will fill the CodeBuild logs otherwise

Reply:
Done